tests/unit/test_cli.py

```
@@ -223,6 +225,65 @@ def test_cli_context_window_override_environment_variable() -> None:
+        assert config.port == 9100
+        assert config.anthropic_port == 9101
+
+
+def test_apply_cli_args_preserves_explicit_anthropic_port() -> None:
+    """Explicit Anthropic port configuration must not be overridden."""
+
+    def fake_load_config(
+        _config_path: Any | None = None,
+        *,
+        resolution: ParameterResolution | None = None,
+        environ: Any | None = None,
```

Ruff (0.13.3)

Unused function argument: `environ`

(ARG001)